ttnn/cpp/ttnn/operations/matmul/device/matmul_op.cpp

```diff
@@ -19,6 +19,8 @@
 using namespace tt::constants;
 using ttnn::operations::unary::UnaryWithParam;
 
+const bool tt::operations::primary::Matmul::enable_stagger = std::getenv("TT_ENABLE_MATMUL_STAGGER") != nullptr && std::string(std::getenv("TT_ENABLE_MATMUL_STAGGER")) == "1";
```

Comment:
Just `std::getenv("TT_ENABLE_MATMUL_STAGGER")` should be fine. Also, please move this inside `add_stagger_defines_if_needed`. There is no reason to propagate this flag to `Matmul` op and all the program functions.

Reply:
Updated; also updated the CI links in the description